apps/api/src/server/app/inspector/inspector.routes.js

```diff
@@ -100,7 +100,7 @@ router.post(
 	// TODO: replace this with an error thrown from `transitionState` else the
 	// route has to know about the intended state transition when that's the
 	// controller's responsibility
-	validateStateTransition('appeal_decided'),
+	validateAppealState('decision_due'),
```

Comment:
so i changed this because the tests were failing for the endpoint to issue a decision. The code for the function `validateStateTransition` was only looking at the current state of the appeal, therefore I changed the name